src/components/rated.js

```diff
@@ -0,0 +1,69 @@
+import Stars from "./stars.js";
+import { Link } from "react-router-dom"
+import React from 'react';
+import { GET_RATED_COCKTAILS } from '../constants';
+import AppDispatcher from '../core/dispatcher';
+import cocktailStore from '../stores/cocktail_store';
+
+export default class Rated extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      cocktails: [],
+      isLoaded: false
+    }
+  }
+
+  loadCocktails = () => {
+    let cocktails = cocktailStore.cocktails;
+    this.setState({
+      cocktails: cocktails,
+      isLoaded: true
+    });
+  }
+
+  componentDidMount() {
+    cocktailStore.on(GET_RATED_COCKTAILS, this.loadCocktails);
+
+    AppDispatcher.dispatch({
+      action: GET_RATED_COCKTAILS,
+      emitOn: [{
+        store: cocktailStore,
+        ids: [GET_RATED_COCKTAILS]
+      }]
+    });
+  }
+
+  componentWillUnmount() {
+    cocktailStore.removeListener(GET_RATED_COCKTAILS, this.loadCocktails);
+  }
+
+  render() {
+    if (!this.state.isLoaded) {
+      return <div className="font-playfair font-normal text-3xl text-center mx-56">Loading...</div>;
+    } else if(this.state.cocktails.length === 0 && this.state.isLoaded){
+      return <div className="font-playfair font-normal text-3xl text-center mx-56">No rated cocktails</div>;
```

Comment:
If they have no rated cocktails, should we maybe provide a link back to the onboard page here?

Reply:
added link to onboard page